### PR TITLE
Enhance e2e slack notification

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

This PR enhances E2E tests Slack notification:

image

[ticket](https://dt-rnd.atlassian.net/browse/K8S-9395)

### CODE REVIEW

File: .github/workflows/e2e-tests.yaml [hunk 1]
```diff
@@ -61,20 +61,19 @@ jobs:
...
     runs-on: ubuntu-latest
+    if: ${{ always() }}
     steps:
       - name: Notify failure in Slack
         uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
         with:
           payload: |
             {
-              "message": ":x: E2E tests failed on ${{ github.event.inputs.target || 'main' }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})",
+              "message": "e2e tests ${{needs.run-in-k8s.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on :kubernetes:\ne2e tests ${{needs.run-in-ocp.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on :openshift:\n using ${{ github.event.inputs.target || 'main' }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})",
```
Comment (albertogdd) [inline .github/workflows/e2e-tests.yaml:76]:
```suggestion
              "message": "E2E tests ${{needs.run-in-k8s.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on :kubernetes:\nE2E tests ${{needs.run-in-ocp.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on :openshift:\n using ${{ github.event.inputs.target || 'main' }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})",
```

Comment (andriisoldatenko) [reply .github/workflows/e2e-tests.yaml]: fixed in 3b723008

[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply .github/workflows/e2e-tests.yaml]: fixed in d1d2be46

[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply .github/workflows/e2e-tests.yaml]: unfortunately all these changes didn't work

**Review by albertogdd — DISMISSED** [review]

**Review by StefanHauth — DISMISSED** [review]

**Review by albertogdd — APPROVED** [review]

File: .github/workflows/e2e-tests.yaml [hunk 2]
```diff
@@ -61,20 +61,19 @@ jobs:
...
     runs-on: ubuntu-latest
+    if: ${{ always() }}
     steps:
       - name: Notify failure in Slack
         uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
         with:
           payload: |
             {
-              "message": ":x: E2E tests failed on ${{ github.event.inputs.target || 'main' }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})",
+              "message": "E2E tests ${{needs.run-in-k8s.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on :kubernetes:\nE2E tests ${{needs.run-in-ocp.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on :openshift:\n using ${{ github.event.inputs.target || 'main' }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})",
```
Comment (chrismuellner) [inline .github/workflows/e2e-tests.yaml:76]: I would change it from
> E2E tests failed on kubernetes
to
> kubernetes failed

E2E test is already in the title and should be pretty obvious

Comment (andriisoldatenko) [reply .github/workflows/e2e-tests.yaml]: @chrismuellner fixed 90fc3971

**Review by chrismuellner — APPROVED** [review]

**Review by albertogdd — APPROVED** [review]

**Pull request merged**